### DISCUSSION TITLE
How to remove Grub theme

### DISCUSSION THREAD

**Discussion by MaxStirnerSpookbuster** [shvchk/fallout-grub-theme#25] [Q&A] [answered]

**Comment by Zorrototo** [comment]
Remove the files if this is what you're talking about, and/or reconfigure you grub config file so it doesn't use the theme.

**Comment by 343GuiltySpark-04** [comment]
might be able to use the grub customize  program too

**Comment by shvchk** [comment, marked as answer]
- Comment out `GRUB_THEME=...` in `/etc/default/grub`
- Update GRUB. This depends on your distro, search for "update grub [distro]".
  Common ways:
    - Debian, Ubuntu, Solus and derivatives: `update-grub` or `grub-mkconfig -o /boot/grub/grub.cfg`
    - RHEL, CentOS, Fedora, SUSE and derivatives: `grub2-mkconfig -o /boot/grub2/grub.cfg`
    - Arch, Gentoo and derivatives: `grub-mkconfig -o /boot/grub/grub.cfg`
